prebid-mobile/pbm-api/android/pbm-targeting-android.md

```diff
@@ -807,18 +792,6 @@ Pay attention that there are certain protected fields such as `regs`, `device`,
 - App and User first party data should use the [functions defined for those purposes](/prebid-mobile/pbm-api/ios/pbm-targeting-ios.html#first-party-data)
 - See the [Prebid Server auction endpoint](/prebid-server/endpoints/openrtb2/pbs-endpoint-auction.html#prebid-server-ortb2-extension-summary) reference for more information about how it will process incoming fields.
 
-### Deprecated 
```

Comment:
Android and iOS docs differ; is it expected? Check out [pbm-targeting-ios.md](https://github.com/prebid/prebid.github.io/pull/5982/files#diff-9f8435b6f9eae45c26aebbb760f32f40ba82a4d1638f15cb494786f856bc30a3R589)